src/utils/localStorage.js

```diff
@@ -5,4 +5,8 @@ export function setOrdersDataToStorage(data) {
 export function getOrdersDataToStorage() {
     const orders = localStorage.getItem('Orders');
     return JSON.parse(orders);
+}
+
+export function clearLocalStorage() {
+    localStorage.clear();
```

Comment:
you should remove only order. Because if you had other data you would delete all data in localStorage